salt/_modules/metalk8s_backups.py

```diff
@@ -0,0 +1,16 @@
+"""Module to manage Metalk8s backups."""
+
+__virtualname__ = "metalk8s_backups"
+
+
+def __virtual__():
+    return __virtualname__
+
+
+def get_backups_to_delete(backup_folder, retention_count):
+    """Return a list of backups to delete."""
+    backups = []
+    ret = __salt__["cmd.run_all"](f'ls -t "{backup_folder}"/*.tar.gz')
+    if ret["retcode"] == 0:
```

Comment:
Why do you use `cmd.run_all` here ? You can do it easily in Python directly or using `file.search` from salt

Reply:
Initially because I wanted files sorted by mtime in a simple way. 
I guess I can do that in python